SETUP.md

```diff
@@ -2,195 +2,107 @@
-eval "$(jenv init -)"
-jenv add /Library/Java/JavaVirtualMachines/<java8 version here>/Contents/Home
-```
-
-#### Install Android Dev Tools
-
-Install the Android SDK and platform tools:
-
-```bash
-brew cask install android-sdk
-brew cask install android-platform-tools
-```
-
-Next install [Android Studio][android studio] and add the [Android NDK][android ndk]
-
-Execute the following (and make sure the lines are in your `~/.bash_profile`):
-
-```bash
-export ANDROID_HOME=/usr/local/share/android-sdk
-export ANDROID_NDK=/usr/local/share/android-ndk
-export ANDROID_SDK_ROOT=/usr/local/share/android-sdk
-export GRADLE_OPTS='-Dorg.gradle.daemon=true -Dorg.gradle.parallel=true -Dorg.gradle.jvmargs="-Xmx4096m -XX:+HeapDumpOnOutOfMemoryError"'
-```
-
-Then install the Android 28 platform:
+We use Yarn to build all of the [celo-monorepo] repo. Install it using [Homebrew](#homebrew):
 
 ```bash
-sdkmanager 'platforms;android-28'
+brew install yarn
 ```
 
-#### Install iOS Dev Tools
-
-Install [Xcode 10.3](https://download.developer.apple.com/Developer_Tools/Xcode_10.3/Xcode_10.3.xip) (an Apple Developer Account is needed to access this link).
-(If after signing in the the direct link does not work try finding on https://developer.apple.com/download/more/)
-
-We do not recommend installing Xcode through the App Store as it can auto update and become incompatible with our projects (until we decide to upgrade).
-
-Note that using the method above, you can have multiple versions of Xcode installed in parallel by using different app names. For instance `Xcode10.3.app` and `Xcode11.app` inside the `/Applications` folder.
-
-Install Cocopods `gem install cocoapods`
-
-From ios folder in mobile package run `bundle exec pod install`
-
 ### Linux
 
 #### Install Node, Yarn and friends
 
-We use Yarn to build all of the [celo-monorepo] repo.
-
-Install `nvm` (allows you to manage multiple versions of Node), Node 8, Node 10 and `yarn`:
+Install NodeJS:
 
 ```bash
-# Installing Node
+# Installing NodeJS
 sudo apt-get update
 sudo apt-get install nodejs
+```
 
-# Installing Nvm
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
-source ~/.bashrc
+Install `nvm` (allows you to manage multiple versions of Node) by following the instructions here: [https://github.com/nvm-sh/nvm].
+
+Once `nvm` is successfully installed, restart the terminal and run the following commands to install the `npm` versions and typescript package that [celo-monorepo] will need:
 
-# Setting up the right version of Nvm
+```bash
+# restart the terminal after installing nvm
 nvm install 8
 nvm install 10
 nvm alias default 10
-
-# Installing Yarn - https://yarnpkg.com/en/docs/install#debian-stable
-curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-sudo apt-get update && sudo apt-get install yarn
-
-# Install typescript
 npm install -g typescript
```

Comment:
Pretty sure we don't require a global typescript installation, not sure why this line is here

Reply:
I'm not sure either but am missing the historical context. I'm good with removing it if don't think we need it.

Reply:
Yeah lets remove it